server/libs/tokenlibs.js

```diff
@@ -0,0 +1,27 @@
+const Token = require('../DataBase/models/tokens');
+const {statusCodes, messages, secretKeys, timeScale} = require ('../utilities/constants');
+
+const createToken = function (token, it) {
+    return new Promise((resolve, reject) => {
+        Token.create({
+            token,
+            createdAt : it,
+            expire_at : it + timeScale.twoDays    
+        }).then((token) => {
+            if (token) {
+                resolve(token);
```

Comment:
no message no status no data??

Reply:
This is a generic function. This is not a function for routing. So that's why only sending back the token which is created.